core/lib/types/src/api/mod.rs

```diff
@@ -637,7 +637,7 @@ pub struct CallTracerConfig {
 #[serde(rename_all = "camelCase")]
 pub struct TracerConfig {
     pub tracer: SupportedTracers,
-    pub tracer_config: CallTracerConfig,
+    pub tracer_config: Option<CallTracerConfig>,
```

Comment:
I'd suggest to annotate this field with `#[serde(default)]` and derive `Default` with `CallTracerConfig`. If I understand correctly, this will achieve the same result without needing to change `debug` namespace implementation.

Reply:
Thanks! Will look into it.